obs/project.go

```diff
@@ -91,3 +97,102 @@ type RepositoryPath struct {
 	Project    string `json:"project" xml:"project,attr"`
 	Repository string `json:"repository" xml:"repository,attr"`
 }
+
+func (c Client) CreateProject(project *Project) error {
```

Comment:
Because this API endpoint is used for updating project as well, we might want to call this function something like `CreateUpdateProject`